internal/label/pullrequests.go

```diff
@@ -13,64 +12,52 @@ import (
+	teamMembers, err := github2.GetTeamMemberList(githubClient, internalTeam)
+	if err != nil {
+		return fmt.Errorf("error getting team members: %w", err) // Properly handle and return error if team member list fetch fails
+	}
+
+	pullRequests, err := github2.FindCommunityPRs(cfg.LabelCheckRepos, teamMembers, githubClient)
+	if err != nil {
+		return fmt.Errorf("error finding community PRs: %w", err) // Handle error from finding community PRs
+	}
+
+	for _, pullRequest := range pullRequests {
+		user := *pullRequest.User.Login
+		if cfg.LabelSkipMap[user] {
+			continue
 		}
-		owner := parts[0]
-		repo := parts[1]
-		pullRequests, err := github2.FindCommunityPRs(owner, repo, teamMembers, githubClient)
-		if err != nil {
-			return err
+		var label string
+		if teamMembers[user] {
+			label = cfg.LabelInternal
+		} else {
+			label = cfg.LabelExternal
 		}
 
-		for _, pullRequest := range pullRequests {
-			if *pullRequest.Number < fullRepo.MinimumNumber {
-				continue
-			}
-			if *pullRequest.Draft {
-				continue
-			}
-			user := *pullRequest.User.Login
-			if cfg.LabelSkipMap[user] {
-				continue
-			}
-			var label string
-			if teamMembers[user] {
-				label = cfg.LabelInternal
-			} else {
-				label = cfg.LabelExternal
+		if label != "" {
+			log.Printf("Pull Request %d by %s will be labeled %s\n", *pullRequest.Number, user, label)
+			hasLabel := false
+			for _, existingLabel := range pullRequest.Labels {
+				if *existingLabel.Name == label {
+					log.Println("Already labeled, skipping...")
+					hasLabel = true
+					break
+				}
 			}
 
-			if label != "" {
-				log.Printf("Pull Request %d by %s will be labeled %s\n", *pullRequest.Number, user, label)
-				hasLabel := false
-				for _, existingLabel := range pullRequest.Labels {
-					if *existingLabel.Name == label {
-						log.Println("  Already labeled, skipping...")
-						hasLabel = true
-						break
-					}
+			if !hasLabel {
+				allLabels := []string{label}
+				for _, labelP := range pullRequest.Labels {
+					allLabels = append(allLabels, *labelP.Name)
 				}
-
-				if !hasLabel {
-					allLabels := []string{label}
-					for _, labelP := range pullRequest.Labels {
-						allLabels = append(allLabels, *labelP.Name)
-					}
-					_, _, err := githubClient.Issues.AddLabelsToIssue(context.TODO(), owner, repo, *pullRequest.Number, allLabels)
-					if err != nil {
-						return fmt.Errorf("error adding labels to pull request %d: %w", *pullRequest.Number, err)
-					}
+				_, _, err := githubClient.Issues.AddLabelsToIssue(context.TODO(), *pullRequest.Base.Repo.Owner.Login, *pullRequest.Base.Repo.Name, *pullRequest.Number, allLabels)
+				if err != nil {
+					return fmt.Errorf("error adding labels to pull request %d: %w", *pullRequest.Number, err) // Ensure error from label adding is handled
 				}
 			}
 		}
 	}
 
-	return nil
+	return nil // Ensure that a nil is returned if the function completes without error
```

Comment:
This doesn't really need a comment imo. Seems pretty clear what's happening here, the comment is fluff.

Reply:
Removed